source/linux/io_uring_context.cpp

```diff
@@ -531,12 +531,16 @@ void io_uring_context::execute_pending_local() noexcept {
 
   LOG("processing local queue items");
 
+#ifdef LOGGING_ENABLED // prevent -Wunused-but-set-variable
   size_t count = 0;
+#endif
   auto pending = std::move(localQueue_);
   while (!pending.empty()) {
     auto* item = pending.pop_front();
     item->execute_(item);
+#ifdef LOGGING_ENABLED
     ++count;
+#endif
```

Comment:
This would be less verbose (and, I think, less error prone); does it work just as well?
```suggestion
  [[maybe_unused]] size_t count = 0;
  auto pending = std::move(localQueue_);
  while (!pending.empty()) {
    auto* item = pending.pop_front();
    item->execute_(item);
    ++count;
```